src/context/VernaContextProvider.tsx

```diff
@@ -196,14 +174,14 @@ function VernaContextProvider({
         fullSchema,
         fullUiSchema,
         handleSubmit,
-        readOnly,
+        isEditor,
+        objectFieldTemplate: { ...defaultObjectFieldTemplate, ...defaultVernaObjectFieldTemplate },
```

Comment:
`defaultObjectFieldTemplate` should be called `objectFieldTemplate` nope?
```suggestion
        objectFieldTemplate: { ...objectFieldTemplate, ...defaultObjectFieldTemplate },
```